src/query/storage/converter.go

```diff
@@ -278,28 +278,32 @@ func iteratorToTsSeries(
 	iter encoding.SeriesIterator,
 	enforcer cost.ChainedEnforcer,
 	tagOptions models.TagOptions,
-) (*ts.Series, error) {
+) (*ts.Series, *ts.Exemplars, error) {
 	metric, err := FromM3IdentToMetric(iter.ID(), iter.Tags(), tagOptions)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	datapoints := make(ts.Datapoints, 0, initRawFetchAllocSize)
+	exemplars := make(ts.Exemplars, 0)
 	for iter.Next() {
-		dp, _, _ := iter.Current()
+		dp, _, annotation := iter.Current()
 		datapoints = append(datapoints, ts.Datapoint{Timestamp: dp.Timestamp, Value: dp.Value})
+		if annotation != nil {
```

Comment:
Does this need any parsing to determine if it's indeed an Exemplar?

Reply:
I am not sure yet.. Depends on what Jerome actually puts into the DB, for now just going to leave it as bytes and print the output as a string.